terraform/deployments/github/main.tf

```diff
@@ -0,0 +1,41 @@
+terraform {
+  backend "s3" {}
+
+  required_providers {
+    github = {
+      source  = "integrations/github"
+      version = "~> 4.0"
+    }
+  }
+}
+
+# NOTE: Uses GITHUB_TOKEN env var, an OAuth / Personal Access Token, for auth
+provider "github" {
+  owner = "alphagov"
+}
+
+#
+# Gives repositories access to push OCI images to GOV.UK Production AWS ECR
+# NOTE: AWS_GOVUK_ECR_ACCESS_KEY_ID and AWS_GOVUK_ECR_SECRET_ACCESS_KEY are
+# manually created.
+#
+
+data "github_repositories" "govuk" {
+  # TODO: Add more granular tags to repositories (i.e. `dockerised`, `replatformed`)
```

Comment:
It's not clear what this TODO is intended to accomplish - is it really worth requiring people to set another tag on (almost) every GOV.UK repo (including new ones and the time taken troubleshooting it when someone forgets, etc.)?